doc/src/manual/arrays.md

```diff
@@ -764,9 +786,12 @@ julia> speye(3,5)
   [3, 3]  =  1.0
 ```
 
-The [`sparse()`](@ref) function is often a handy way to construct sparse matrices. It takes as
-its input a vector `I` of row indices, a vector `J` of column indices, and a vector `V` of stored
-values. `sparse(I,J,V)` constructs a sparse matrix such that `S[I[k], J[k]] = V[k]`.
+The [`sparse()`](@ref) function is often a handy way to construct sparse array. For
+matrices it takes as its input a vector `I` of row indices, a vector `J` of column
+indices, and a vector `V` of stored values. `sparse(I,J,V)` constructs a sparse matrix
+such that `S[I[k], J[k]] = V[k]`. The equivalent sparse vector constructor is
+[`sparsevec`](@ref), which takes one vector `I` with indices and one vector `V` with the
```

Comment:
we could maybe deprecate passing scalars and instead flatten out any 1-element array inputs, or possibly something clever with broadcast syntax for that use case, but not sure if it's worth it